recipe/build.bat

```diff
@@ -0,0 +1,67 @@
+@echo off
+
+if not exist Cargo.toml xcopy /E /I /H /Y "%SRC_DIR%\..\..\..\src_cache\test_extract\zed-%PKG_VERSION%\*" "%SRC_DIR%\"
+
+mkdir "%PREFIX%\Menu"
+copy "%RECIPE_DIR%\menu.json" "%PREFIX%\Menu\%PKG_NAME%_menu.json"
+copy "crates\zed\resources\app-icon.png" "%PREFIX%\Menu\zed.png"
+
+set CARGO_PROFILE_RELEASE_STRIP=symbols
+set ZED_UPDATE_EXPLANATION=Please use your package manager to update zed from conda-forge
+set CARGO_TARGET_DIR=C:\b
+
+REM Enable incremental compilation to reduce memory usage
```

Comment:
```suggestion
REM Enable incremental compilation to reduce memory usage
```

I don't think we should build incrementally, there's nothing incremental here after all